### ISSUE TITLE
[Bug]: TreeChart disappears when there is only one connection

### ISSUE THREAD

**Issue by r4dc** [carbon-design-system/carbon-charts#1781]
### Application/Team

N/A

### What happened?

The TreeChart appears for a brief moment and then disappears when there is only one connection.

### Version

```
"@carbon/charts": "^1.15.3",
"@carbon/charts-react": "^1.15.3",
```

### StackBlitz example

https://stackblitz.com/edit/h8pauk?file=data.js,index.js

### What priority level would this be in your opinion?

P3

**Comment by dgantait** [comment]
We also want to have this fix , as our tree chart nodes are dynamic build based on some condition an some cases there might be only once child node. In D3 it was working perfectly as expected, but here the chart with single child node disappeared. Please help us to resolve it.

**Comment by vladimir-kim-interspark** [comment]
I have the same issue and instead of logging a new issue, appending to this.
Issue: When there is a single path for the tree, it disappears:

![Image](https://github.com/user-attachments/assets/11638b9c-e3cb-4051-bff6-03ef92bdafa2)

It is not expected behaviour.
